client/my-sites/checkout/composite-checkout/components/wp-checkout-order-summary.js

```diff
@@ -20,14 +20,26 @@ import {
 } from '@automattic/wpcom-checkout';
 
 /**
+/**const
  * Internal dependencies
  */
-import { isPlan, isMonthly, getYearlyPlanByMonthly, getPlan } from '@automattic/calypso-products';
-import { isJetpackSite } from 'calypso/state/sites/selectors';
+import {
+	isPlan,
+	isMonthly,
+	getYearlyPlanByMonthly,
+	getPlan,
+	isFreePlanProduct,
+} from '@automattic/calypso-products';
+import { isJetpackSite, getSite } from 'calypso/state/sites/selectors';
 import isAtomicSite from 'calypso/state/selectors/is-site-automated-transfer';
 import Gridicon from 'calypso/components/gridicon';
 import getPlanFeatures from '../lib/get-plan-features';
 import { hasDomainCredit } from 'calypso/state/sites/plans/selectors';
+import {
+	TERM_MONTHLY,
+	TERM_ANNUALLY,
+	TERM_BIENNIALLY,
+} from '@automattic/calypso-products/src/constants';
```

Comment:
These should be imported directly from `@automattic/calypso-products` rather than reaching into the `src` directory. The directory structure of an npm package is not typically expected to be part of its public API and could change without warning. It's safer to just rely on the public API. 

```suggestion
} from '@automattic/calypso-products';
```